libraries/standard/http/test/Makefile

```diff
@@ -5,7 +5,7 @@ CSDK_ROOT = ../../../..
 SOURCE = ../src
 
 # Show extra warnings.
-CFLAGS += -Wextra
+CFLAGS += -Wextra -O0 -ggdb
```

Comment:
We are adding more tests using this framework?

Reply:
I primarily added these flags to be able to debug the test binaries with debugging symbols. When this framework is replaced with the Unity/CMock framework, that should also use these flags